### DISCUSSION TITLE
How to refresh/reset/redraw/repaint the screen?

### DISCUSSION THREAD

**Discussion by GlassGruber** [helix-editor/helix#4349] [Q&A] [answered]
I'm not really sure what is the correct term, so I've listed some that came to mind regarding this minor display issue.

I'm a new user and testing things around I've faced some glitches like characters of previous lines not cleared when moving cursor while testing different color themes.  
I think however that this is more related to my terminal rather than an hx direct issue.

So I thought something like using shortcut `ctrl+L` could "refresh" current screen and remove all displayed inconsistencies, but nothing happened and looking for commands like `:redraw` `:clear` came up to nothing (BTW the fuzzy cmd `:` search is very smooth and the tooltip tui pretty nice, hat to you!🎩)

I've checked the [keymap](https://docs.helix-editor.com/keymap.html) docs page but haven't found anything about this. 

Hope it's not a RTFM case, thank you!

**Comment by sudormrfbin** [comment, marked as answer]
There is no way to redraw the screen currently, opening an issue: #4350.

**Reply by GlassGruber** [reply]
Oh nice thank you!
